torch/testing/_internal/common_methods_invocations.py

```diff
@@ -156,15 +177,23 @@ def __init__(self,
         self.handles_extremals = handles_extremals
         self.handles_complex_extremals = handles_complex_extremals
 
+        # Epsilon to ensure grad and gradgrad checks don't test values
+        #   outside a function's domain.
+        self._domain_eps = 1e-5
+
+    def sample_inputs(self, device, dtype, requires_grad=False):
+        low, high = self.domain
+        low = low if low is None else low + self._domain_eps
+        high = high if high is None else high - self._domain_eps
+
+        return (SampleInput(make_tensor((20,), device, dtype,
```

Comment:
Why is the size is 20 here? Should this use one of the constants from below?

Reply:
Sure.